### ISSUE TITLE
[BUG] Undefined variable `Decimal` using 3.1.5

### ISSUE THREAD

**Issue by nordowl** [chrishoermann/zod-prisma-types#212]
**Describe the bug**
Generating the schemas having a `Decimal` field in a Prisma model results in an undefined variable `Decimal` being used.
```ts
export const ModelWhereInputSchema: z.ZodType<Prisma.ModelWhereInput> = z.object({
  // other fields...
  chance: z.union([ z.lazy(() => DecimalFilterSchema),z.union([z.number(),z.string(),z.instanceof(Decimal),z.instanceof(Prisma.Decimal),DecimalJsLikeSchema,]).refine((v) => isValidDecimalInput(v), { message: 'Must be a Decimal' }) ]).optional(),
}).strict();
```
`z.instanceof(Decimal)` throws an error, because `Decimal is undefined`.

I was using zod-prisma-types 2.8.1 up until now, where the following was generated:
```ts
export const ModelWhereInputSchema: z.ZodType<Prisma.ModelWhereInput> = z.object({
  // other fields...
  chance: z.union([ z.lazy(() => DecimalFilterSchema),z.union([z.number(),z.string(),DecimalJSLikeSchema,]).refine((v) => isValidDecimalInput(v), { message: 'Must be a Decimal' }) ]).optional(),
}).strict();
```

**Package versions (please complete the following information):**
 - zod: 3.22.3
 - prisma: 5.5.2

**Additional context**
I think the error was introduced with the 3.0 release. Was using 2.8.1 together with Zod and Prisma as listed above, without any errors.

**Comment by chrishoermann** [comment]
@chrishoermann sorry my bad. Since v. 3.0.0 the generator checks if `decimal.js` is installed and if so adds validation for `Decimal` to the fields. I just forgot a conditional check. I'll release a fix shortly!

**Comment by chrishoermann** [comment]
@nordowl This should now be fixed in `3.1.6`

**Comment by nordowl** [comment]
Awesome, thank you! Can confirm this is fixed now.
I'm closing this issue.
